js/SubmitAllView.js

```diff
@@ -22,10 +22,10 @@ export default class SubmitAllView extends Backbone.View {
 
   render() {
     this.articleView.$el.addClass('no-submit-buttons');
-    const {
-      buttonText,
-      ariaLabel
-    } = Adapt.course.get('_buttons')._submit;
+    const buttons = this.model.get('_button');
+    const courseSubmitButtons = Adapt.course.get('_buttons')._submit;
+    const buttonText = buttons.buttonText || courseSubmitButtons.buttonText;
```

Comment:
This doesn't work if buttons is undefined (cannot lookup property on undefined) or buttons.buttonText is intentionally set to a null string `""` (falsey with or).
```suggestion
    const buttonText = buttons?.buttonText ?? courseSubmitButtons.buttonText;
```